### PR TITLE
Added a KeepBangImportant option to preserve '!important' in final style attribute.

### PR DESCRIPTION
**Pull request by Vindexus**
My final output needed to have `margin: 0 auto !important` in the style attribute to work properly with the template I was using.

I added this option and tests to achieve that.

### CODE REVIEW

File: premailer/element.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ type elementRules struct {
 	element         *goquery.Selection
 	rules           []*styleRule
 	cssToAttributes bool
+	keepBangImportant bool
```
Comment (vanng822) [inline premailer/element.go:17]: Do we need this flag or should we always keep "important"?

Comment (Vindexus) [reply premailer/element.go:17]: I'm not sure actually. I added it as a flag so it wouldn't change functionality for anyone already using the code.

Comment (vanng822) [reply premailer/element.go:17]: Ok, not me either so let's keep it that way

**Comment by vanng822** [comment]
@Vindexus Should I fix the test issue

**Comment by vanng822** [comment]
I do it

**Pull request merged**